docs/releases/3.0.txt

```diff
@@ -131,6 +131,13 @@ Minor features
   The default value is :func:`~django.utils.translation.get_language()` instead
   of :setting:`LANGUAGE_CODE`.
 
+App-loading changes
+~~~~~~~~~~~~~~~~~~~~
+
+* ``django.setup()`` is now idempotent, and its actual operations can be
+  overridden using :setting:`DJANGO_SETUP_CALLABLE`, e.g. to setup test mockups
```

Comment:
It think it's just `test mocks` (removing the need for the `spelling_wordlist` change).